src/index.ts

```diff
@@ -35,7 +37,7 @@ async function getApp(app: App, baseUrl: string) {
   const octokit = new Octokit({
     auth: jwt,
     baseUrl,
-    previews: ['antiope-preview'],
+    previews: ['antiope-preview', ...custom_previews],
```

Comment:
Thoughts on making a variable for this up top so we don't have to repeat in three places?

Reply:
Could do that too!